java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1023,6 +1026,32 @@ public Table orderBy(OrderByArg... args) {
     return new Table(orderBy(nativeHandle, sortKeys, isDescending, areNullsSmallest));
   }
 
+  public static Table merge(List<Table> tables, OrderByArg... args) {
```

Comment:
Missing Javadoc

Reply:
Thanks for catching this. It should be done now.